src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -72,8 +74,14 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
-    def getByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
-      clusterQueryWithLabels.filter { _._1.googleProject === project.string }.filter { _._1.clusterName === name.string }.result map { recs =>
+    def getActiveClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
+      clusterQueryWithLabels.filter { _._1.googleProject === project.string }.filter { _._1.clusterName === name.string }.filter{_._1.destroyedDate === Timestamp.from(Instant.parse(dummyDate))}.result map { recs =>
+        unmarshalClustersWithLabels(recs).headOption
+      }
+    }
+
+    def getDeletingClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
```

Comment:
just checking: this is only used by tests?

Reply:
Yes, I changed the `getByName` to `getActiveClusterByName`, so had to create another one just for the tests to return the deleting cluster. 

Should `getActiveClusterByName` return a deleting cluster?

Reply:
that would be incredibly confusing. i vote no :) it wasn't a criticism, just confirming